`npm install` fail with "Permission denied (publickey)"

I am running `npm install` from the [svg2pdf.js](https://github.com/yWorks/svg2pdf.js/tree/master) folder but it fails with:

```
npm ERR! git clone git@github.com:github:chick307/adler32cs.js Cloning into bare repository '/home/pharmony/.npm/_git-remotes/git-github-com-github-chick307-adler32cs-js-029b2fc0'...
npm ERR! git clone git@github.com:github:chick307/adler32cs.js Permission denied (publickey).
npm ERR! git clone git@github.com:github:chick307/adler32cs.js fatal: Could not read from remote repository.
npm ERR! git clone git@github.com:github:chick307/adler32cs.js
npm ERR! git clone git@github.com:github:chick307/adler32cs.js Please make sure you have the correct access rights
npm ERR! git clone git@github.com:github:chick307/adler32cs.js and the repository exists.
```

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
 	"bugs": "https://github.com/chick307/adler32cs.js/issues",
 	"repository": {
 		"type": "git",
-		"url": "git://github.com/chick307/adler32cs.js.git"
+		"url": "git+https://github.com/chick307/adler32cs.js.git"
 	},
 	"dependencies": {},
 	"devDependencies": {

